.github/workflows/build-and-push-rag-content.yaml

```diff
@@ -46,7 +47,7 @@ jobs:
           registry: registry.redhat.io
 
       - name: Build only partial rag-content when running in the check pipeline
-        if: github.event_name != 'schedule' || github.event_name != 'push'
+        if: github.event_name != 'schedule' && github.event_name != 'push'
```

Comment:
We have only four trigger events in total. Pull request being the only when we don't want run entire build. Wouldn't it be easier to trigger partial build on pull_request?

Reply:
I also wanted to run the partial build on the `workflow_dispatch`. But if we agree that PR is the only trigger we want to use for partial build then I'm fine with it.

Reply:
Resolved:)